crates/flux-middle/src/rty/normalize.rs

```diff
@@ -20,7 +20,7 @@ use crate::{
 #[derive(TyEncodable, TyDecodable)]
 pub struct NormalizedDefns {
     krate: CrateNum,
-    defns: UnordMap<FluxId<DefIndex>, NormalizeInfo>,
+    pub defns: UnordMap<FluxId<DefIndex>, NormalizeInfo>,
```

Comment:
I think you are not accessing this outside
```suggestion
    defns: UnordMap<FluxId<DefIndex>, NormalizeInfo>,
```